codespell_lib/data/dictionary.txt

```diff
@@ -7082,7 +7082,7 @@ contrl->control
 contrller->controller
 contro->control
 controlable->controllable
-controle->control, controlled, controller,
+controle->control, controlled, controls, controller,
```

Comment:
I think it's more likely that the user would mean `controls` than `controlled` — what do you think?
```suggestion
controle->control, controls, controlled, controller,
```